src/components/loginForm/LoginForm.js

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { loginUser } from '../../state/features/userSlice/userSlice';
+
+export default function LoginForm() {
+	let [myUsername, setUsername] = useState('');
+	let [myPassword, setPassword] = useState('');
+	let [isWrong, setWrong] = useState(false);
+
+	const navigate = useNavigate();
+
+	const user = useSelector((state) => state.user.user);
+
+	useEffect(() => {
+		if (user.username) {
+			navigate('/');
+		}
+	}, [user]);
+
+	const dispatch = useDispatch();
+
+	function handleLogin() {
+		fetch('http://localhost:4000/login', {
```

Comment:
create a separate request for this one, no complicated logic inside the `callback`
`function handleLogin() {
      fetchUser(); //this should be implemented separately
}`